src/workspaces/components/AddVocabularyForm.tsx

```diff
@@ -1,23 +1,72 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { useSelector } from 'react-redux'
-import { TextField, Checkbox, FormControlLabel } from '@material-ui/core'
+import {
+  TextField,
+  Checkbox,
+  FormControlLabel,
+  FormControl,
+  FormLabel,
+  RadioGroup,
+  Radio,
+  Box,
+} from '@material-ui/core'
 import t from 'app/components/i18n'
 import useActionForm from 'app/hooks/useActionForm'
 import Actions from 'app/actions'
 import { getIsAddVocabularyFormOpen, getWorkspace } from 'workspaces/selectors'
 import useDispatchAction from 'app/hooks/useDispatchAction'
 import FormDialog from 'app/components/FormDialog'
 
+const vocabularyTypes = [
+  {
+    label: 'Datový',
+    prefix: 'https://slovník.gov.cz/datový/',
+    regex: '^https://slovník.gov.cz/datový/[ěščřžýáíéóúůďťňa-z0-9]+$',
+  },
+  {
+    label: 'Agendový',
+    prefix: 'https://slovník.gov.cz/agendový/',
+    regex: '^https://slovník.gov.cz/agendový/[a-z0-9]+$',
+  },
+  {
+    label: 'Legislativní',
+    prefix: 'https://slovník.gov.cz/legislativní/sbírka/',
+    regex: '^https://slovník.gov.cz/legislativní/sbírka/[0-9]+/[0-9]+$',
+  },
+  {
+    label: 'Generický',
+    prefix: 'https://slovník.gov.cz/generický/',
+    regex: '^https://slovník.gov.cz/generický/[a-z0-9]+$',
```

Comment:
Please change to  (my fault):

 ^https://slovník.gov.cz/generický/[ěščřžýáíéóúůďťňa-z0-9]+$

Reply:
I modified all the regular expressions to also allow "-" as a character in the last part of URI